contracts/test/LON/Setup.t.sol

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.7.6;
+
+import "forge-std/Test.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "contracts/Lon.sol";
+import "contracts-test/utils/BalanceSnapshot.sol";
+
+contract TestLON is Test {
+    using SafeMath for uint256;
+    using SafeERC20 for IERC20;
+    using BalanceSnapshot for BalanceSnapshot.Snapshot;
+
+    uint256 userPrivateKey = uint256(1);
+    uint256 otherPrivateKey = uint256(2);
+
+    address user = vm.addr(userPrivateKey);
+    address spender = address(0x133701);
+    address emergencyRecipient = address(0x133702);
+
+    Lon lon = new Lon(address(this), emergencyRecipient);
+
+    uint256 DEADLINE = block.timestamp + 1;
+
+    struct Permit {
+        address owner;
+        address spender;
+        uint256 value;
+        uint256 nonces;
+        uint256 deadline;
+    }
+    Permit DEFAULT_PERMIT;
```

Comment:
Permit seems to be used only in `permit` test. Maybe code related to `permit` could be collected in permit test contract?

Reply:
No problem, I will use the setUp() inheritance way to downsize Setup.t.sol and move permit struct to Permit.t.sol file in the next commit